onnxruntime/test/framework/ort_model_only_test.cc

```diff
@@ -236,7 +236,39 @@ static void DumpOrtModelAsJson(const std::string& model_uri) {
   std::ofstream(model_uri + ".json") << json;
 }
 */
+/* The full build was causing the following error because the graph node array has some empyt (blan) node at some indices for certain ORT designs
+       onnx runtime exception : Satisfied, but should not be : node == nullptr
```

Comment:
Would be better to document why it happened vs. the effect of the issue. Bug was due to loading an ORT format model in a full build, allowing optimizers to run, but trying to use the saved kernel information. Because the optimizers removed nodes you get a mismatch.